flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcRowDataLookupFunctionTest.java

```diff
@@ -82,6 +103,121 @@ void testLookup(boolean withFailure) throws Exception {
         assertThat(result).isEqualTo(expected);
     }
 
+    @ParameterizedTest
+    @MethodSource("lookupWithPredicatesProvider")
+    public void testEval(TestSpec testSpec) throws Exception {
```

Comment:
```suggestion
    void testEval(TestSpec testSpec) throws Exception {
```
no need for `public` since it is junit5